.github/dependabot.yaml

```diff
@@ -146,6 +146,20 @@ updates:
       # requires to also bump `TEST_INFRA_DEFINITIONS_BUILDIMAGES` inside `.gitlab/common/test_infra_version.yml`
       # and dependabot isn’t able to keep those two synchronized.
       - dependency-name: github.com/DataDog/test-infra-definitions
+      # Do not bump dependencies that come from test-infra-definitions, they will be bumped when we'll bump the version
+      # https://datadoghq.atlassian.net/browse/ADXT-375
+      - dependency-name: github.com/Masterminds/semver
+      - dependency-name: github.com/alessio/shellescape
+      - dependency-name: github.com/cenkalti/backoff/v4
+      - dependency-name: github.com/google/gofuzz
+      - dependency-name: github.com/aws/*
+      - dependency-name: github.com/pulumi/*
+      - dependency-name: github.com/pulumiverse/pulumi-time/sdk
+      - dependency-name: github.com/samber/lo
+      - dependency-name: github.com/stretchr/testify
+      - dependency-name: golang.org/x/exp
+      - dependency-name: gopkg.in/yaml.v3
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Wondering if we should limit to dependencies that cause plugins downloads, so `pulumi*` ones. Other dependencies are safe to bump, even if we should keep them in sync

Reply:
That's a good point. Let me change this!